docs/source/bp/network_segmentation.rst

```diff
@@ -0,0 +1,49 @@
+..
+  Created by: mike garcia
+  To: network segmentation
+
+.. |bp_title| replace:: Segment Networks Based on Sensitivity
+
+|bp_title|
+----------------------------------------------
+
+Network Segmentation is the practice of splitting a network into multiple sub-networks. These networks are usually designed around business needs, for example, having sub-networks for executives, finance, operations, and human resources or by keeping election functions separated from other county functions. Networks can also be separated by data sensitivity, keeping more sensitive election data separated from every day communications.
+
+Keeping information segmented can shrink the attack surface: successful attacks on one part of a network are less likely to impact the other parts. It can also lead to performance improvements within each sub-network.
+
+Network segmentation can be physical or logical. Physical segmentation keeps network traffic separate through devices such as firewalls and switches. Logical segmentation uses virtual networks and addressing schemes to keep traffic on its intended sub-network. 
```

Comment:
Additional segmentation can be achieved through network isolation, where an entirely separate network is created. This is often performed in the field of elections for sensitive election data like tabulators. There are often other portions of the election management system that may be connected to an in isolated network. Additionally, organizations may wish to segregate high-risk applications from the general network. For instance, multiple employees may need to access applications used to design ballots, and they all may need to push and pull data from the same election datastore (e.g., fileserver).